frontend/src/data/utils/Constants.ts

```diff
@@ -1,26 +1,131 @@
-// Constant strings to be used in tests and elsewhere
-
-// TODO add values for the other categories. Maybe convert to an enum.
+// MULTIUSE TERMS
 export const ALL = "All";
 export const TOTAL = "Total";
+export const UNKNOWN = "Unknown";
+
+// RACE AND ETHNICITY DEMOGRAPHIC TERMS
 export const RACE = "race_and_ethnicity";
+
+// select, specific vars
 export const WHITE_NH = "White (Non-Hispanic)";
 export const WHITE = "White";
 export const ASIAN_NH = "Asian (Non-Hispanic)";
 export const HISPANIC = "Hispanic or Latino";
 export const NON_HISPANIC = "Not Hispanic or Latino";
 export const TWO_OR_MORE = "Two or more races";
+export const UNKNOWN_RACE = "Unknown race";
+export const UNKNOWN_HL = "Unknown Hispanic or Latino";
+export const UNKNOWN_ETHNICITY = "Unknown ethnicity";
 
+// GROUPED RACE DEMOGRAPHICS
+export const STANDARD_RACES = [
+  "American Indian and Alaska Native (Non-Hispanic)",
```

Comment:
It feels a little bit weird that only some of these are constants. Maybe we should take the time to come up with a better way of defining all of the demographic groups on the frontend and stick to it. I'm fine merging this for now so we dont have to keep thinking about merge conflicts but this would be a good thing to think about I think

Reply:
Those old individually declared ones were there already for tests; I guess we should take those out and then just type check our tests? 

```
export const WHITE_NH = "White (Non-Hispanic)";
export const WHITE = "White";
export const ASIAN_NH = "Asian (Non-Hispanic)";
export const HISPANIC = "Hispanic or Latino";
export const NON_HISPANIC = "Not Hispanic or Latino";
```

Reply:
I just fixed it up a little; basically declare all race options as consts and then combine then into arrays; this allows us to use the single consts in tests / checks throughout the app but also keeps them organized into arrays for the filtering we do in the menu, etc. I didn't bother to export all of the age groups since we don't use any individual age group options in the code. The one that had been declared for a test i moved into the test file and used the new `AgeBucket` type